app/modules/academics/repository.py

```
@@ -9,6 +9,9 @@ def get_by_id(self, school_id: int, year_id: int): return self.db.query(models.A
     def create(self, school_id: int, data: schemas.AcademicYearCreate):
         if data.is_current: self.db.query(models.AcademicYear).filter(models.AcademicYear.school_id == school_id).update({models.AcademicYear.is_current: False})
         db_year = models.AcademicYear(**data.model_dump(), school_id=school_id); self.db.add(db_year); self.db.commit(); self.db.refresh(db_year); return db_year
+    def update(self, year: models.AcademicYear, data: schemas.AcademicYearUpdate):
+        for key, value in data.model_dump(exclude_unset=True).items(): setattr(year, key, value.strip() if isinstance(value, str) else value)
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)